Code/Comscore/Source/integration/ComscoreConfiguration.swift

```diff
@@ -11,6 +11,7 @@ public class ComScoreConfiguration {
     public let publisherId: String
     public let applicationName: String
     public var userConsent: ComScoreUserConsent
+    public var usagePropertiesAutoUpdateMode: ComscoreUsagePropertiesAutoUpdateMode
```

Comment:
Maybe I don't see the complete picture, but:
if this value can't change (e.g. if we change it, but the underlying analytics plugin is already initialized with the old value (and it never updates)) consider making it a `let`, or we need to make sure the underlying code takes into account the updated value

Reply:
✅